Add EL9 links to the nightly upgrade guide (#3046)
theforeman/foreman-documentation: guides/common/modules/proc_upgrading-smartproxy-server.adoc

For information on backups, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.

ifdef::foreman-el,katello[]
. On {EL} 9: Update repositories:
+
[options="nowrap" subs="attributes"]
----
ifdef::katello[]
# {project-package-update} https://yum.theforeman.org/releases/{ProjectVersion}/el9/x86_64/foreman-release.rpm \
https://yum.theforeman.org/katello/{KatelloVersion}/katello/el9/x86_64/katello-repos-latest.rpm
endif::[]
ifdef::foreman-el[]
# {project-package-update} https://yum.theforeman.org/releases/{ProjectVersion}/el9/x86_64/foreman-release.rpm
endif::[]
----

> . On {EL} 8: Update repositories:

Reviewer: Similarly here.

# dnf module disable pulpcore
----
endif::[]

> . On {EL} 8: Switch to the PostgreSQL 13 module:

Reviewer 2: Pre-existing problem, but this part should be inside `ifdef::katello[]` since PostgreSQL isn't on a Smart Proxy with vanilla Foreman. Only with Katello.

Author: I moved the endif::[] to include this in the katello section.

> . Switch to the PostgreSQL 13 module if running {EL} 8:

Reviewer 3: ```suggestion
. On {EL} 8: Switch to the PostgreSQL 13 module:
```

same for below.
